packages/commitlint-issue-reference/example/index.ts

```diff
@@ -0,0 +1,221 @@
+import { readFileSync, writeFileSync } from 'node:fs';
+import processCommitMessage, { LintOptions } from '../src/index.js';
+
+// return absolute path to fixture file
+function getFixturePath(fileName: string): string {
+  const fixtureDir = './fixtures';
+  const fixturePath = `${fixtureDir}/${fileName}`;
+  // make absolute
+  return new URL(fixturePath, import.meta.url).pathname;
+}
+
+function runTest({
+  fileName,
+  ...options
+}: { fileName: string } & Partial<Omit<LintOptions, 'file'>>) {
+  const filePath = getFixturePath(fileName);
+
+  // save current file content, to restore after the test
+  const fileContent = readFileSync(filePath, 'utf8');
+
+  processCommitMessage({
+    file: filePath,
+    ...options,
+  });
+
+  writeFileSync(filePath, fileContent);
+}
+
+// HEADER
+
+// correct
```

Comment:
No, i'm using this to uncomment and test specific cases. Since the process currently terminates the process with an exit code, i don't have an easy method if running all of them at the same time :)

Reply:
I added an explanation comment above it.